examples/typescript/src/index.html

```diff
@@ -0,0 +1,87 @@
+<!DOCTYPE html>
```

Comment:
The idea is to mainly move the GitHub pages from source code to avoid confusion and clean the repository a bit.

I'm not entirely sure how a good example approach will be here but moving it all to a bundle with parcel which should be testable both locally and in GitHub pages.